package.json

```
@@ -47,7 +47,7 @@
     "typecheck": "tsc -p tsconfig.json"
   },
   "dependencies": {
-    "@clevercloud/client": "11.0.1",
+    "@clevercloud/client": "11.1.0",
```

fix: wrong version number 

```suggestion
    "@clevercloud/client": "11.1.1",
```